README.md

```diff
@@ -37,9 +37,9 @@
 - Create a `components` directory inside the `src` directory, and then create a sub-directory called `SearchBar`, another one called `PostContainer`, and lastly one called `CommentSection`, all inside the `src` directory. Each of these directories should contain the component file as well as the CSS for their respective components. You'll also want to put any other components that coincide with your respective container components inside of these directories.
```

Comment:
Need to clean up files.

Noticed you have an instaclone, instagram, and instagram-app folder.

Delete instagram and instagram-app folders.